src/serialize.h

```diff
@@ -490,12 +490,12 @@ class Wrapper
 template<typename Formatter, typename T>
 static inline Wrapper<Formatter, T&> Using(T&& t) { return Wrapper<Formatter, T&>(t); }
 
-#define VARINT(obj, ...) Using<VarIntFormatter<__VA_ARGS__>>(obj)
+#define VARINT(obj, mode) Using<VarIntFormatter<mode>>(obj)
```

Comment:
Would suggest something like:

```c++
#define VARINT(obj)            VARINT_MODE(obj, VarIntMode::DEFAULT)
#define VARINT_MODE(obj, mode) Using<VarIntFormatter<mode>>(obj)
```

to reduce verbosity and number of changes needed here

Reply:
Done.